Orbitersdk/samples/ProjectApollo/src_sys/yaAGC/agc_engine.c

```diff
@@ -1847,7 +1847,7 @@ SimulateDV(agc_t *State, uint16_t divisor)
 // it requires slightly more processing, in order to conform as obviously
 // as possible to the MIT docs.
 
-#define SCALER_OVERFLOW 80
+#define SCALER_OVERFLOW 20
```

Comment:
We should not be diverging from VirtualAGC, it will only cause problems and frustration in the future. If a change to yaAGC is needed a PR should be made in https://github.com/virtualagc/virtualagc.

Reply:
See my general response above. I'm open to start with modifying first VirtualAGC - actually even I find it the neatest way - but then please help me out with a method on how to transform the VirtualAGC-compatible agc_engine to NASSP-compatible one for testing. Maybe creating a patch between the two "official" versions in NASSP and VAGC, and applying it to my developments will do the job... But I'm really curious on what is the best practice here.